### ISSUE TITLE
Ember orange fails color contrast against white text - suggested color

### ISSUE THREAD

**Issue by j-afarian** [ember-learn/ember-styleguide#21]
The existing ember orange **#E04E39** fails minimum color contrast against the white foreground text (as is already indicated in the README).

Just wanted to make a suggestion for **#D14632** which is visually extremely close, but passes WCAG 2.0 AA criteria [1.4.3 Contrast (Minimum)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html) with a ratio of 4.53:1 against white text.

## White on orange
![Color contrast analyzer: #FFFFFF against #D14632](https://s17.postimg.org/64m4wuz5r/D14632_accessible.png)

## Comparison of existing color vs new color
![Color contrast analyzer: #E04E39 against #D14632](https://s17.postimg.org/kb1vs8mxb/comparison.png)

**Comment by MelSumner** [comment]
Thank you for filing this issue, we appreciate it! 

Since it passes if the font is large enough(18.66px and bold or 24px if not bold), we use it only in that way. For other instances, we do have additional variants of Ember orange ($orange-darker, $orange-darkest) and those pass the necessary color contrast checks.

Screenshot of Color Contrast Checker: 
![image](https://user-images.githubusercontent.com/4587451/36263315-956d6b2a-122f-11e8-8d7e-285b536f2615.png)

**Comment by MelSumner** [comment]
Adding: https://github.com/ember-learn/ember-styleguide/pull/23 updates the project readme to more accurately reflect the current status. Thanks again for your input! :)
